<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotogamePage</class>
 <widget class="QWidget" name="MotogamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Watch replay</string>
   </property>
   <widget class="QPushButton" name="watchButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BlockIndex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Block#</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlockCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Mine</string>
   </property>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image: url(:/images/play_button);
border:none;
}
QPushButton:hover
{
image: url(:/images/play_button_hover);
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>If you need help press &lt;a href=&quot;mine-help&quot;&gt;here&lt;/a&gt;.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QRadioButton" name="radioLowQ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioHighQ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rendering:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioOGL3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenGL 3</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSoft">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Software</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
